scripts/buildorder.py

```diff
@@ -179,16 +189,27 @@ def __init__(self, subpackage_file_path, parent, virtual=False):
             raise Exception("SubPackages should have a parent")
 
         self.name = os.path.basename(subpackage_file_path).split('.subpackage.sh')[0]
-        if "gpkg" in subpackage_file_path.split("/")[-3].split("-") and "glibc" not in self.name.split("-"):
+        if "gpkg" in subpackage_file_path.split("/")[-3].split("-") and not has_prefix_glibc(self.name):
             self.name = add_prefix_glibc_to_pkgname(self.name)
         self.parent = parent
-        self.deps = set([parent.name])
-        self.only_installing = parent.only_installing
+        self.virtual = virtual
+        self.only_installing = None if virtual else parse_build_file_variable(subpackage_file_path, "TERMUX_SUBPKG_ONLY_INSTALLING")
```

Comment:
`TERMUX_SUBPKG_ONLY_INSTALLING` is not used anywhere else, and is introduced in this PR. If it is not needed anywhere I don't think it should be added.